nym-vpn-lib/src/platform/android.rs

```diff
@@ -173,11 +172,57 @@ pub extern "system" fn Java_net_nymtech_vpn_NymVpnService_defaultTunConfig<'env>
     env: JNIEnv<'env>,
     _this: JObject<'_>,
 ) -> JObject<'env> {
+    use jnix::IntoJava;
     let env = JnixEnv::from(env);
-
     TunConfig::default().into_java(&env).forget()
 }
 
+impl<'env> IntoJava<'env> for String {
+    type JavaType = JString<'env>;
+
+    fn into_java(self, env: &JNIEnv<'env>) -> Self::JavaType {
+        env.new_string(&self).expect("Failed to create Java String")
+    }
+}
+
+pub trait IntoJava<'env> {
```

Comment:
Let's make a comment here about that ideally we'd use the IntoJava trait in jnix and it's impl for String, but that we didn't get it to work so this is a quick fix for now. In the future maybe we can remove this trait and use the jnix directly